README.rst

```diff
@@ -0,0 +1,154 @@
+
+   .. raw:: html
+
+      <script language="javascript">
+      var version = 'latest';
+      function fillCitation(){
+         $('#tedana_version').text(version);
+
+         function cb(err, zenodoID) {
+            getCitation(zenodoID, 'vancouver-brackets-no-et-al', function(err, citation) {
+               $('#tedana_citation').text(citation);
+            });
+            getDOI(zenodoID, function(err, DOI) {
+               $('#tedana_doi_url').text('https://doi.org/' + DOI);
+               $('#tedana_doi_url').attr('href', 'https://doi.org/' + DOI);
+            });
+         }
+
+         if(version == 'latest') {
+            getLatestIDFromconceptID("1250561", cb);
+         } else {
+            getZenodoIDFromTag("1250561", version, cb);
+         }
+      }
+      </script>
+      <p>
+      <span id="tedana_citation">tedana</span> Available from: <a id="tedana_doi_url" href="https://doi.org/10.5281/zenodo.1250561">https://doi.org/10.5281/zenodo.1250561</a>
+      <img src onerror='fillCitation()' alt=""/>
+
+      <p>
+      2. Kundu, P., Inati, S. J., Evans, J. W., Luh, W. M. & Bandettini, P. A. (2011).
+      <a href=http://dx.doi.org/10.1016/j.neuroimage.2011.12.028>Differentiating BOLD and non-BOLD signals in fMRI time series using multi-echo EPI.</a>
+      <i>NeuroImage</i>, <i>60</i>, 1759-1770.
+      </p>
+      <p>
+      3. Kundu, P., Brenowitz, N. D., Voon, V., Worbe, Y., Vértes, P. E., Inati, S. J.,
+      Saad, Z. S., Bandettini, P. A., & Bullmore, E. T. (2013).
+      <a href=http://dx.doi.org/10.1073/pnas.1301725110>Integrated strategy for improving functional connectivity mapping using multiecho fMRI.</a>
+      <i>Proceedings of the National Academy of Sciences</i>, <i>110</i>, 16187-16192.
+      </p>
+
+Alternatively, you can automatically compile relevant citations by running your
+tedana code with `duecredit`_. For example, if you plan to run a script using
+tedana (in this case, ``tedana_script.py``):
+
+.. code-block:: bash
+
+ python -m duecredit tedana_script.py
+
+You can also learn more about `why citing software is important`_.
+
+.. _Differentiating BOLD and non-BOLD signals in fMRI time series using multi-echo EPI.: http://dx.doi.org/10.1016/j.neuroimage.2011.12.028
+.. _Integrated strategy for improving functional connectivity mapping using multiecho fMRI.: http://dx.doi.org/10.1073/pnas.1301725110
+.. _duecredit: https://github.com/duecredit/duecredit
+.. _`why citing software is important`: https://www.software.ac.uk/how-cite-software
+
+Installation
+------------
+
+You'll need to set up a working development environment to use ``tedana``.
+To set up a local environment, you will need Python >=3.6 and the following
+packages will need to be installed:
+
+- mdp
+- nilearn
+- nibabel>=2.1.0
+- numpy
+- scikit-learn
+- scipy
+
+You can then install ``tedana`` with:
+
+.. code-block:: bash
+
+  pip install tedana
+
+Getting involved
+----------------
+
+We :yellow_heart: new contributors!
```

Comment:
RST doesn't support emojis, unfortunately ☹️. We can swap this out with `<3` or "love" or something, though.

Reply:
I think it works if you just copy and paste in the emoji (eg: from https://emojipedia.org/yellow-heart) (based on [this stackoverflow answer](https://stackoverflow.com/a/42256239)) could you give that a bash @tsalo and see if it works? Otherwise I'm totally fine with `<3` as you suggested!

Reply:
It works! Thanks!